tests/test_rp_vars.py

```diff
@@ -53,4 +53,23 @@ def test_get_all_rp_vars(new_solver_session_no_transcript) -> None:
     # case reader comparison
     case = CaseReader(case_filepath=case_path)
     case_vars = case.rp_vars()
-    assert len(case_vars) == pytest.approx(9000, 450)
\ No newline at end of file
+    assert len(case_vars) == pytest.approx(9000, 450)
+
+
+@pytest.mark.dev
+@pytest.mark.fluent_232
+def test_rp_vars_allowed_values(new_solver_session_no_transcript) -> None:
+    solver = new_solver_session_no_transcript
+    rp_vars = solver.rp_vars
+
+    assert rp_vars("number-of-iterations") == 0
+
+    with pytest.raises(RuntimeError) as msg:
+        rp_vars("number-of-iterat")
+
+    assert msg.value.args[0] == "number-of-iterat is not an allowed  name.\n" \
```

Comment:
noting the double space. Is this because you are not specifying the argument.  

We should pass the argument e.g., "variable".

Also the implementation of allowed_name_error_message should check the string to decide if the extra space is needed

Reply:
Done, Thank you :)